### PR TITLE
Bugfix/fixes for data table

### PR DESCRIPTION
**Pull request by hannyle**
### Description

Fixed for folder's Options/Rights in data table, fixed for table's sorting functionality, and modified modals to have similar width.

### Related issues
Fixes https://github.com/CSCfi/swift-browser-ui/issues/801, https://github.com/CSCfi/swift-browser-ui/issues/815, https://github.com/CSCfi/swift-browser-ui/issues/803, https://github.com/CSCfi/swift-browser-ui/issues/808, https://github.com/CSCfi/swift-browser-ui/issues/816

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### Changes Made

- Checked **accessRights** of folders and greyed out some table's options according to folder's Shared status
- Fixed for sorting functionality for table's **Name** and **Items** columns
- Modified modals to have the same width

### Testing

- [ ] Unit Tests
- [ ] Integration Tests
- [x] Tests do not apply

### CODE REVIEW

File: swift_browser_ui_frontend/src/common/conv.js [hunk 1]
```diff
@@ -279,24 +279,30 @@ export function tokenize(text, ignoreSmallerThan = 2) {
...
 export function sortObjects(objects, sortBy, sortDirection) {
-  sortBy = sortBy === "size" ? "bytes" : sortBy;
+  sortBy = sortBy === "size" ? "bytes"
+    : sortBy === "items" ? "count" : sortBy;
 
   objects.sort((a, b) => {
     let valueA = a[sortBy];
     let valueB = b[sortBy];
 
+
```
Comment (sampsapenna) [inline swift_browser_ui_frontend/src/common/conv.js:289]: Minor styling thing, but could we possibly have a tad fewer empty lines here?

**Review by sampsapenna — CHANGES_REQUESTED** [review]

File: swift_browser_ui_frontend/src/common/conv.js (hunk 1 above)
Comment (hannyle) [reply swift_browser_ui_frontend/src/common/conv.js]: Yes, redundant empty lines are removed. Thanks!

**Review by sampsapenna — CHANGES_REQUESTED** [review]
Needs rebasing before merge, otherwise LGTM 👍🏼

**Comment by hannyle** [comment]
> Needs rebasing before merge, otherwise LGTM 👍🏼

It is now rebased on latest `devel` branch

**Review by sampsapenna — APPROVED** [review]

**Pull request merged**